.travis.yml

```diff
@@ -1,15 +1,24 @@
 language: node_js
 node_js:
   - "6"
+
 script:
   - yarn lint
   - yarn test
+
 before_deploy:
   - yarn build
+
 deploy:
-  provider: npm
-  email: $NPM_EMAIL
-  api_key: $NPM_KEY
-  skip_cleanup: true
-  on:
-    tags: true
+  - provider: npm
+    email: $NPM_EMAIL
+    api_key: $NPM_KEY
+    skip_cleanup: true
+    on:
+      tags: true
+  - provider: pages
+    github-token: $GITHUB_TOKEN
+    local-dir: styleguide
+    skip-cleanup: true
+    on:
+      branch: master
```

Comment:
This will allow to publish content of the `./styleguide` on push to master, right?

But could you help me understand, will `build:docs` be triggered as well, to populate this dir? Or how that would work?

Also, could you post, what would be the expected URL?

Thanks!

Reply:
`build:docs` generates html, js, css in `./styleguide` directory on every change of master.

`provider: pages` pushes the content of this directory to `gh-pages` branch and it gets published automatically by github.

Current URL is https://bblfsh.github.io/uast-viewer/

To change it to the custom domain we need to add DNS CNAME to point to github and set it in repository settings. (I mentioned it in the issue)